recipes/robotpy-cscore/conda_build_config.yaml

```diff
@@ -0,0 +1,8 @@
+cudnn:                                    # [linux64]
```

Comment:
I think you don't need this file see https://conda-forge.org/docs/maintainer/knowledge_base.html?#using-centos-7

The `conda-forge.yml` file doesn't exist until the feedstock is made. It might need to be fixed in the feedstock after this is merged.

Reply:
Heh, I guess this bitrotted during the last few months. Let me know if you want me to rebase this.